src/libratbag-data.c

```diff
@@ -442,6 +450,26 @@ file_data_matches(struct ratbag *ratbag,
 		}
 	}
 
+	data->type = g_key_file_get_string(keyfile, GROUP_DEVICE, "DeviceType", NULL);
+	log_debug(ratbag, "Device Type: '%s'\n", data->type == NULL ? "no type found" : data->type);
+
+	if(data->type == NULL) {
+		// Good idea? Or just add a DeviceType Property to everything
```

Comment:
I think it's fine for now, eventually we don't want this since we have control over all device files.